src/pages/Home/FeaturedPhotos.js

```diff
@@ -66,11 +66,14 @@ const FeaturedText = styled.p`
   font-size: 0.825rem;
   line-height: 1.36em;
   font-weight: 300;
+  @media ${device.mobile} {
+    font-size: 11px;
+  }
   @media ${device.tablet} {
     font-size: 1.5rem;
   }
   @media ${device.laptop} {
-    font-size: 2.375rem;
+    font-size: 2rem;
```

Comment:
Same Q here about font size changing - I'm assuming this was to make the layout work? I'm wondering if changing the font-size is the right solution to this problem though. Perhaps we want to change around some spacing or size of divs instead?

Reply:
ah, there you go! see my previous comment.